sdk/src/main/java/com/sygic/travel/sdk/di/sygicAuthApiModule.kt

```diff
@@ -0,0 +1,36 @@
+package com.sygic.travel.sdk.di
+
+import com.github.salomonbrys.kodein.Kodein
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.instance
+import com.github.salomonbrys.kodein.singleton
+import com.google.gson.Gson
+import com.sygic.travel.sdk.auth.api.SygicTravelAuthApiClient
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+
+internal val sygicAuthApiModule = Kodein.Module {
+	bind<OkHttpClient>("sygicAuthHttpClient") with singleton {
+		val builder = OkHttpClient.Builder()
+
+		if (instance("isInDebugMode")) {
+			builder.addInterceptor(instance<HttpLoggingInterceptor>())
+		}
+		builder.build()
+	}
+
+	bind<Retrofit>("sygicAuthApiRetrofit") with singleton {
+		Retrofit.Builder()
+			.client(instance("sygicAuthHttpClient"))
+			.baseUrl("https://tripomatic-auth-master-testing.sygic.com")
```

Comment:
this is alpha URL, everywhere should be a production url.

we will make it configurable later.

Reply:
OK, but when I switch to production URL no method from `AuthFacade` will authenticate you. All of them returns 401 code. I presume, that it is because in production all of them requires `client_secret` field.

Reply:
Discuss this with @costasovo.